src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visitbalances/VisitBalanceMigrationResource.kt

```diff
@@ -171,4 +171,44 @@ class VisitBalanceMigrationResource(
     ],
   )
   suspend fun activeMigration() = migrationHistoryService.getActiveMigrationDetails(MigrationType.VISIT_BALANCE)
+
+  @PostMapping("/{migrationId}/refresh")
```

Comment:
it does seem weird that this isn't in a generic migration resource rather than having to be added for each migration separately.

Reply:
I suspect there's no strategic reason for this - there's still a lot of stuff in this project (and the UI) which is generic and could be consolidated.